test/Npgsql.Tests/BatchTests.cs

```diff
@@ -288,6 +288,18 @@ public async Task StatementOID()
         Assert.That(batch.BatchCommands[1].OID, Is.EqualTo(0));
     }
 
+    [Test]
+    public void CanCreateParameter()
+    {
+        Assert.True(new NpgsqlBatchCommand().CanCreateParameter);
```

Comment:
nit: expression-bodied method (and below)

Reply:
OK.